### ISSUE TITLE
gff2gtf creates duplicated value

### ISSUE THREAD

**Issue by CFeregrino** [NBISweden/AGAT#258]
Hello, thank you for the wonderful package. I have been using it, like many others, to prepare GTFs for scATAC-seq workflows. Having using it succesfully before, I now have problems running gff2gtf on a custom-extended GTF. 
The problem is that a get a duplicated value in a specific attribute. As you can see below, the attribute exon_numer has two values for some reason. This seems to be incompatible with the workflows I am using (CellRanger). Is there a way to fix it?

I'm using AGAT v0.9.0 on Conda

Original gtf:
```
MT      .       exon    1       68      1000    +       .       gene_id "ENSMUST00000082387.1"; exon_number "1"; exon_number_from_end "1"; transcript_id "ENSMUST00000082387.1"; gene_name "mt-Tf"; exon_id "ENSMUSE00000521514.1"; gene_type "Mt_tRNA"; level "3"; mgi_id "MGI:102487"; transcript_name "mt-Tf-201"; exon_number "1"; tag "basic"; transcript_type "Mt_tRNA"; transcript_support_level "NA";
MT      .       exon    70      1024    1000    +       .       gene_id "ENSMUST00000082388.1"; exon_number "1"; exon_number_from_end "1"; transcript_id "ENSMUST00000082388.1"; gene_name "mt-Rnr1"; exon_id "ENSMUSE00000521515.1"; gene_type "Mt_rRNA"; level "3"; mgi_id "MGI:102493"; transcript_name "mt-Rnr1-201"; exon_number "1"; tag "basic"; transcript_type "Mt_rRNA"; transcript_support_level "NA";
MT      .       exon    1025    1093    1000    +       .       gene_id "ENSMUST00000082389.1"; exon_number "1"; exon_number_from_end "1"; transcript_id "ENSMUST00000082389.1"; gene_name "mt-Tv"; exon_id "ENSMUSE00000521516.1"; gene_type "Mt_tRNA"; level "3"; mgi_id "MGI:102472"; transcript_name "mt-Tv-201"; exon_number "1"; tag "basic"; transcript_type "Mt_tRNA"; transcript_support_level "NA";
MT      .       exon    1094    2675    1000    +       .       gene_id "ENSMUST00000082390.1"; exon_number "1"; exon_number_from_end "1"; transcript_id "ENSMUST00000082390.1"; gene_name "mt-Rnr2"; exon_id "ENSMUSE00000521517.1"; gene_type "Mt_rRNA"; level "3"; mgi_id "MGI:102492"; transcript_name "mt-Rnr2-201"; exon_number "1"; tag "basic"; transcript_type "Mt_rRNA"; transcript_support_level "NA";
MT      .       exon    2676    2750    1000    +       .       gene_id "ENSMUST00000082391.1"; exon_number "1"; exon_number_from_end "1"; transcript_id "ENSMUST00000082391.1"; gene_name "mt-Tl1"; exon_id "ENSMUSE00000521518.1"; gene_type "Mt_tRNA"; level "3"; mgi_id "MGI:102482"; transcript_name "mt-Tl1-201"; exon_number "1"; tag "basic"; transcript_type "Mt_tRNA"; transcript_support_level "NA";
```
Output gtf:
```
##gtf-version 3
MT      .       gene    1       68      1000    +       .       gene_id "ENSMUST00000082387.1"; transcript_id "ENSMUST00000082387.1"; ID "ENSMUST00000082387.1"; exon_id "ENSMUSE00000521514.1"; exon_number "1" "1"; exon_number_from_end "1"; gene_name "mt-Tf"; gene_type "Mt_tRNA"; level "3"; mgi_id "MGI:102487"; tag "basic"; transcript_name "mt-Tf-201"; transcript_support_level "NA"; transcript_type "Mt_tRNA";
MT      .       transcript      1       68      1000    +       .       gene_id "ENSMUST00000082387.1"; transcript_id "ENSMUST00000082387.1"; ID "ENSMUST00000082387.1"; Parent "ENSMUST00000082387.1"; exon_id "ENSMUSE00000521514.1"; exon_number "1" "1"; exon_number_from_end "1"; gene_name "mt-Tf"; gene_type "Mt_tRNA"; level "3"; mgi_id "MGI:102487"; original_biotype "rna"; tag "basic"; transcript_name "mt-Tf-201"; transcript_support_level "NA"; transcript_type "Mt_tRNA";
MT      .       exon    1       68      1000    +       .       gene_id "ENSMUST00000082387.1"; transcript_id "ENSMUST00000082387.1"; ID "ENSMUSE00000521514.1"; Parent "ENSMUST00000082387.1"; exon_id "ENSMUSE00000521514.1"; exon_number "1" "1"; exon_number_from_end "1"; gene_name "mt-Tf"; gene_type "Mt_tRNA"; level "3"; mgi_id "MGI:102487"; tag "basic"; transcript_name "mt-Tf-201"; transcript_support_level "NA"; transcript_type "Mt_tRNA";
MT      .       gene    70      1024    1000    +       .       gene_id "ENSMUST00000082388.1"; transcript_id "ENSMUST00000082388.1"; ID "ENSMUST00000082388.1"; exon_id "ENSMUSE00000521515.1"; exon_number "1" "1"; exon_number_from_end "1"; gene_name "mt-Rnr1"; gene_type "Mt_rRNA"; level "3"; mgi_id "MGI:102493"; tag "basic"; transcript_name "mt-Rnr1-201"; transcript_support_level "NA"; transcript_type "Mt_rRNA";
MT      .       transcript      70      1024    1000    +       .       gene_id "ENSMUST00000082388.1"; transcript_id "ENSMUST00000082388.1"; ID "ENSMUST00000082388.1"; Parent "ENSMUST00000082388.1"; exon_id "ENSMUSE00000521515.1"; exon_number "1" "1"; exon_number_from_end "1"; gene_name "mt-Rnr1"; gene_type "Mt_rRNA"; level "3"; mgi_id "MGI:102493"; original_biotype "rna"; tag "basic"; transcript_name "mt-Rnr1-201"; transcript_support_level "NA"; transcript_type "Mt_rRNA";
```

**Comment by Juke34** [comment]
I'm not 100% sure but I guess this is due to identical isoforms that are automatically merged together, since v0.9 we merge the attributes in such cases. I a future release it would be possible to handle this behaviour.
I will try to investigate more in details

**Comment by CFeregrino** [comment]
Oh! Sorry that I didn't inspected my input thoroughly
Thank you for your help, anyways, it really fixed my problem although it had nothing to do with your package.
